package-testing/pre/000_install_package.rb

```diff
@@ -7,12 +9,28 @@
       scp_to(workstation, ENV['LOCAL_PKG'], pkg)
     end
 
-    if workstation['platform'] =~ %r{windows}
+    case workstation['platform']
+    # TODO: BKR-1109 requests a supported way to install packages on Windows and OSX
+    when /windows/
       pkg ||= "http://#{ENV['BUILD_SERVER']}/pdk/#{ENV['SHA']}/repos/windows/pdk-x64.msi"
-
-      # BKR-1109 requests a neater way to install an MSI
       generic_install_msi_on(workstation, pkg)
+
+    when /osx/
+      unless pkg
+        # TODO: Replace this code that fetches build yaml and slices details from it. Is there a way to know the URL to the dmg file, given only SHA? 
+        build_yaml = open("http://#{ENV['BUILD_SERVER']}/pdk/#{ENV['SHA']}/repos/#{ENV['SHA']}.yaml").read
+        build_yaml = YAML.load(build_yaml)
+        artifact = build_yaml[:platform_data][workstation['platform']][:artifact][1..-1]
+        pkg = "http://#{ENV['BUILD_SERVER']}/pdk/#{ENV['SHA']}/repos#{artifact}"
```

Comment:
I think Jenkins provides the required build string here as SUITE_VERSION.
If I use that then I can remove this yaml and string slicing code.
That would mean one more ENV for a local user to have to manage; but that's better than having to rely on this code...